sos/report/plugins/__init__.py

```diff
@@ -1453,28 +1462,79 @@ def add_copy_spec(self, copyspecs, sizelimit=None, maxage=None,
         if isinstance(tags, str):
             tags = [tags]
 
+        def get_filename_tag(fname):
+            """Generate a tag to add for a single file copyspec
+
+            This tag will be set to the filename, minus any extensions
+            except '.conf' which will be converted to '_conf'
+            """
+            fname = fname.replace('-', '_')
+            if fname.endswith('.conf'):
+                return fname.replace('.', '_')
```

Comment:
This will for `fname = "foo.bar.conf"` return `foo_bar_conf` - are *both* / all replacements intentional, or just the latest one?